internal/store/postgres/organization_repository.go

```diff
@@ -201,7 +201,25 @@ func (r OrganizationRepository) List(ctx context.Context, flt organization.Filte
 			"id": goqu.Op{"in": flt.IDs},
 		})
 	}
-	query, params, err := stmt.ToSQL()
+	offset := flt.Pagination.Offset()
+	limit := flt.Pagination.PageSize
+
+	totalCountStmt := stmt.Select(goqu.COUNT("*"))
+	totalCountQuery, _, err := totalCountStmt.ToSQL()
+	if err != nil {
+		return []organization.Organization{}, fmt.Errorf("%w: %s", queryErr, err)
+	}
+
```

Comment:
By default, does this method have any order by clause? If not, we should have one by `created_at DESC` since we're introducing pagination.
This is so that the API response is easier to consume and understand, otherwise we could get a different result each time since we're now returning paginated results.

Reply:
Added sorting on `created_at` DESC.